group.go

```diff
@@ -436,3 +446,36 @@ func (g *Group) Delete(recursive bool) error {
 	}
 	return nil
 }
+
+// AddMemberWithType adds a member to the Group providing its type
```

Comment:
Was this comment cut off? 🤔 Either way, I would add a period.

Reply:
Done